config/environments/test.rb

```diff
@@ -35,12 +41,23 @@
   # Print deprecation notices to the stderr.
   config.active_support.deprecation = :stderr
 
+  # Raises error for missing translations
+  # config.action_view.raise_on_missing_translations = true
+
+  config.action_mailer.default_url_options = { host: 'localhost:3000' }
+
+  
+  # Configure static asset server for tests with Cache-Control for performance.
+  config.serve_static_files = true
+  config.static_cache_control = 'public, max-age=3600'
+
   # Fake omniauth for testing
   OmniAuth.config.test_mode = true
 
   config.after_initialize do
+    # TODO: re-enable and go through failures
```

Comment:
@asideofcode-dev can we create a Github issue for this? We want to make sure we remember to do it.

Reply:
Done, here you go https://github.com/codebar/planner/issues/1885

Reply:
Thanks :)